src/Json.Schema.ToDotNet.UnitTests/DataModelGeneratorTests.cs

```diff
@@ -871,10 +871,43 @@ public void GeneratesCloningCode()
       ""type"": ""integer"",
       ""description"": ""An integer property.""
     },
+    ""intPropWithDefault"": {
```

Comment:
>intPropWithDefault [](start = 6, length = 18)

I find the jschema test strategy unsatisfying in that there's no clear marker in tests of the real world impact of changes. Perhaps you could add some single practical example of the framework in use that would illustrate more effectively how updates alter code emit. TBD